docs/src/docs/asciidoc/user/tiering.adoc

```diff
@@ -88,7 +92,7 @@ include::{sourcedir310}/ehcache-impl/src/test/java/org/ehcache/docs/Tiering.java
 ----
 
 <1> This will limit the amount of memory used by the heap tier for storing key-value pairs.
-There is a cost associated with sizing objects.
+There is a cost associated with sizing objects. Byte based heap resources are deprecated and will be removed in a future version.
```

Comment:
I think this sentence becomes excessive if we have a warning at the start of the section.

Reply:
Agreed, Incorporated